packages/cli/src/commands/project/list.ts

```diff
@@ -27,7 +27,7 @@ export default async function list(
 
   output.spinner(`Fetching projects in ${chalk.bold(contextName)}`);
 
-  let projectsUrl = `/v4/projects/?limit=20`;
+  let projectsUrl = `/v9/projects?limit=20`;
```

Comment:
this is quite the api version bump

Reply:
A lengthy explanation 🙃

The `vc domain inspect` function was using an even older version of the endpoint (v2) which was not a paginated version of the endpoint, yet that command did its own version of pagination based on the most recent project's `updatedAt` field.

At some point, this endpoint (all versions) stopped sorting results upon the `updatedAt` field (specifically when the `domain` query param was used, which was indeed the case for `domain inspect`). This caused the pagination code in the CLI command to enter an endless loop, because it would keep jumping around with various timestamps because it was seeing duplicate results. We never saw this in the tests before because the issue would only happen for a team with > 50 accounts, which was never the case with the previous test accounts.

On top of that, the `domain` query param does even work for the GET projects anymore (all versions), so there's no way to get the API to return the projects specific to a domain name. Thus, the `domain` param was removed from CLI (so that the results get sorted by `updatedAt` so that pagination works), and the filtering by relevant domain logic was implemented client-side.

In the case of this file, `vc project list` was also updated to use the same version of the API endpoint for consistency.